lib/ssz_ex.ex

```diff
@@ -428,6 +428,21 @@ defmodule LambdaEthereumConsensus.SszEx do
     (size + @bits_per_chunk - 1) |> div(@bits_per_chunk)
   end
 
+  @doc """
+  Returns the default value for a schema, which can be a basic or composite type.
+  """
+  def default({:int, _}), do: 0
+  def default(:bool), do: false
+  def default({:bytes, size}), do: <<0::size(size * 8)>>
+  def default({:list, _, _}), do: []
+  def default({:vector, inner_type, size}), do: default(inner_type) |> List.duplicate(size)
+  def default({:bitlist, _}), do: BitList.default()
+  def default({:bitvector, size}), do: BitVector.new(size)
+
+  def default(module) when is_atom(module) do
+    module.schema() |> Enum.map(fn {_attr, schema} -> default(schema) end)
```

Comment:
maybe we could add a test in ssz static that creates a `default` container and encodes it to ensure it doesn't error?

Or maybe it's a bit overkill and we can do it in ssz generic instead

Reply:
We can add it as a check in `LambdaEthereumConsensus.Container.__after_compile__/2`, just to see if it fails anywhere.